composer.json

```diff
@@ -17,12 +17,17 @@
          "php": ">=5.6.0"
      },
     "require-dev": {
-        "phpunit/phpunit": ">=5.5.0",
-        "mikey179/vfsStream": "~1"
+        "phpunit/phpunit": "^5.5",
```

Comment:
- To avoid latest PHPUnit version installed and unexpected issues, it should let `PHPUnit` install with `5.5.x` versions.

To fix following message, it should be `mikey179/vfsstream`.

```
Deprecation warning: require-dev.mikey179/vfsStream is invalid, it should not contain uppercase characters. Please use mikey179/vfsstream instead. Make sure you fix this as Composer 2.0 will error.
```